relayer/spy_relayer/src/__tests__/validation.ts

```diff
@@ -0,0 +1,186 @@
+// createCommonEnvironment and createListenerEnvironment get invoked, so set these env variables
+process.env.LOG_LEVEL = "debug";
```

Comment:
Yeah I agree it's not the cleanest. I tried moving it to a `beforeAll` hook but since those two imports are compiled first, these tests fail to run. Have to put it here 😞